pkgs/applications/misc/appmenu-gtk-module/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, fetchurl, cmake, glib, gtk2, gtk3 }:
+let
+xinitfile = fetchurl {
+      url = "https://raw.githubusercontent.com/archlinux/svntogit-community/packages/appmenu-gtk-module/trunk/80-appmenu-gtk-module";
+      sha256 = "1g4zjqnr89achb8k97j26f0dl8wy7afm4dbdmmq60mdqlx76q02c";
+};
+in
+stdenv.mkDerivation {
+  pname = "appmenu-gtk-module";
+  version = "0.7.3.2";
+  src = fetchFromGitHub {
+    owner = "rilian-la-te";
+    repo = "vala-panel-appmenu";
+    rev = "0.7.3.2";
+    fetchSubmodules = true;
+    sha256 = "0fp8d38sjxpx8xpq0hxm67p7clhc3cbag581srxba6rhdcj5brkd";
+  };
+  sourceRoot = "source/subprojects/appmenu-gtk-module";
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ glib gtk2 gtk3 ];
+
+   cmakeFlags = [
+    "-DGTK3_INCLUDE_GDK="
+  ];
+
+  installPhase = ''
+    make install
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Port of the Unity GTK Module";
+    license = licenses.lgpl3Only;
```

Comment:
https://gitlab.com/vala-panel-project/vala-panel-appmenu/-/issues/331

Reply:
Let's see the outcome of this.